src/index.spec.js

```diff
@@ -1202,6 +1208,7 @@ describe('Dropzone', () => {
         </Dropzone>
       )
       await dropzone.simulate('drop', createDtWithFiles(files))
+      await flushPromises(dropzone)
```

Comment:
Question: why is this needed and can’t we put it into `afterEach` for example?

Reply:
I think it was introduced by you or someone else. Without it, tests will fail. It's probably because of the file aggregation which happens async.

Reply:
Yes it’s needed because of async get items. But the question still stands: can we remove the repetition and put it into `afterEach`?

Reply:
Will try it and update if it does.

Reply:
Nope, it does not work if I move `flushPromises()` to `afterEach`.

Reply:
@okonet so, what do you think, should we merge this?